packages/xo-server-perf-alert/src/index.js

```diff
@@ -523,12 +523,14 @@ ${monitorBodies.join('\n')}`
                 }\n`
 
                 return result
-              } catch (_) {
+              } catch (error) {
+                console.warn('xo-server-perf-alert:', error)
                 return {
                   uuid,
                   object: null,
                   objectLink: `cannot find object ${uuid}`,
                   listItem: `  * ${uuid}: **Can't read performance counters**\n`,
+                  error,
```

Comment:
For what reason did you put the error here?

Where does it end up? Where can it be seen?

Reply:
You're right, I removed it.